dev/releases/make_archives.py

```diff
@@ -205,11 +205,17 @@
     with open(filename+".sha256", 'w') as file:
         file.write(sha256file(filename))
 
+    # now create the file package-infos.json
+    subprocess.run(["../bin/gap.sh", "-r", "--quitonbreak",
```

Comment:
But at this point we already deleted all packages. Plus we never compiled the JSON package... so how can this work?

Reply:
Hm, I must have tested this with the GAP in my repositories root directory which has JSON compiled. :/

Reply:
I think I just pushed a fix. I'm testing it now.

Reply:
Ok, I have finally figured out where to compile json and call the new function. Let's see.